assets/styles/elements/_normalize.scss

```diff
@@ -0,0 +1,100 @@
+// ==========================================================================
+// Elements / Normalize
+// ==========================================================================
+
+// Modern CSS Normalize
+// Based on the reset by Andy.set with some tweaks.
+// Original by Andy.set: https://piccalil.li/blog/a-more-modern-css-reset/
+// Review by Chris collier: https://chriscoyier.net/2023/10/03/being-picky-about-a-css-reset-for-fun-pleasure/
+
+
+// Box sizing rules
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+// Prevent font size inflation
+html {
+    -moz-text-size-adjust: none;
+    -webkit-text-size-adjust: none;
+    text-size-adjust: none;
+}
+
+// Remove default margin in favour of better control in authored CSS
+p,
+h1,
+h2,
+h3,
+h4,
+dl,
+dd,
+body,
+figure,
+blockquote {
+    margin-block-end: 0;
+}
+
+// Remove list styles on ul, ol elements with a list role, which suggests default styling will be removed
+ul[class],
+ol[class] {
```

Comment:
This comment is referring to Bell's use of `[role="list"]`. This should be updated to reflect `[class]`. I presume this was based on [Coyier's suggestion](https://chriscoyier.net/2023/10/03/being-picky-about-a-css-reset-for-fun-pleasure/).

Reply:
Yep, comment updated !